charts/argo/values.yaml

```diff
@@ -226,6 +226,11 @@ server:
     #   kubernetes.io/ingress.class: nginx
     #   kubernetes.io/tls-acme: "true"
 
+    ## Labels to be added to the web ingress.
+    ##
+    # labels:
```

Comment:
Should we name it additionalLabels to clearly indicate that it won’t replace the default labels? What do you think @stefansedich @vladlosev?

Reply:
In this chart, we already have multiple instances of the `podLabels` and `serviceLabels` attributes specifying _additional_ labels on different object. Same goes for additional annotation attributes. So leaving this attribute as just `labels` will be consistent with the existing convention.